### PR TITLE
Initial commit#2

### PR DESCRIPTION
**Pull request by plankton9991**
- [DEMO LINK](https://plankton9991.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://plankton9991.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,66 @@
...
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  /* height: 60px; */
```
Comment (illiarerun) [inline src/style.css:14]: don't leave commented code
```suggestion
```

File: src/style.css [hunk 2]
```diff
@@ -1 +1,66 @@
...
+}
+
+.nav__link {
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  text-transform: uppercase;
+  color: #000;
+
+  margin-left: 20px;
+  padding: 23px 0 19px;
```
Comment (illiarerun) [inline src/style.css:41]: links shouldn't have horizontal paddings, it should be a margin for list items

File: src/style.css [hunk 3]
```diff
@@ -1 +1,66 @@
...
+}
+
+.is-active::after {
+  content: "";
+  display: block;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 8px;
+  transform: translateY(19px);
```
Comment (illiarerun) [inline src/style.css:65]: bottom: 0?

File: src/style.css [hunk 4]
```diff
@@ -1 +1,66 @@
...
+  margin-left: 0;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
```
Comment (illiarerun) [inline src/style.css:56]: combine these selectors

**Review by illiarerun — CHANGES_REQUESTED** [review]
Good job, now let's work on fixes:

**Review by MaksimusCRS — APPROVED** [review]

**Pull request closed without merging**